File: Learning/OtherLanguages/C/C练习题/Use C/1.c

void main()
{
    int i, j, k, count=0;
    for(i=1; i<5; i++)
    {
        for(j=1; j<5; j++)
        {
            for(k=1; k<5; k++)
            {
                if(i!=j && i!=k && j!=k)
                {
                    printf("%d%d%d\n", i, j, k);
                    count++;
                }
            }
        }
    }
    printf("Total:%d\n", count);
}

File: Learning/OtherLanguages/C/C练习题/Use Python/1.py
count = 0
for i in range(1,5):
    for j in range(1,5):
        for k in range(1,5):
            if i != j and i != k and j != k:
                print(i, j, k)
                count += 1
print('共有%d个数' % count)

                
